quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildAndStartDistTest.java

```diff
@@ -17,69 +17,48 @@
 
 package org.keycloak.it.cli.dist;
 
-import java.util.function.Consumer;
 import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
-import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestMethodOrder;
-import org.keycloak.it.junit5.extension.BeforeStartDistribution;
 import org.keycloak.it.junit5.extension.CLIResult;
 import org.keycloak.it.junit5.extension.DistributionTest;
 import org.keycloak.it.junit5.extension.RawDistOnly;
 import org.keycloak.it.utils.KeycloakDistribution;
 
-import io.quarkus.test.junit.main.Launch;
-import io.quarkus.test.junit.main.LaunchResult;
-
 import static org.keycloak.quarkus.runtime.cli.command.AbstractStartCommand.OPTIMIZED_BUILD_OPTION_LONG;
 
-@DistributionTest(reInstall = DistributionTest.ReInstall.NEVER)
+@DistributionTest
 @RawDistOnly(reason = "Containers are immutable")
 @TestMethodOrder(OrderAnnotation.class)
 public class BuildAndStartDistTest {
 
     @Test
-    @Launch({ "build", "--cache=local" })
-    @Order(1)
-    void testBuildWithCliArgs(LaunchResult result) {
-        CLIResult cliResult = (CLIResult) result;
+    void testBuildAndStart(KeycloakDistribution dist) {
+        // start using based on the build options set via CLI
+        CLIResult cliResult = dist.run("build", "--storage=chm");
```

Comment:
Do we need to be hardcoding `--storage=chm`? Can't we somehow leverage the default options coming from `ServerOptions` class?

This applies to other test classes changed in this PR as well.